### PR TITLE
use new image service

### PR DESCRIPTION
**Pull request by JakeChampion**
We launched the new image service 6 days ago, I'm currently going around FT projects and helping them migrate to it.

The new Image Service includes a lot of improvements:
- The application is multi-region with failover
- We use stale-on-error and stale-while-revalidate cache extensions to ensure users get stale content if the application errors or has to revalidate an image
- We serve much smaller WebP and JPEG XR images to supported browsers
- We’ve seen a reduction in file size for a large number of image requests
- We’re now serving images over H2

### CODE REVIEW

File: client/src/assets/sass/components/content.scss [hunk 1]
```diff
@@ -72,7 +72,7 @@
             padding: 32px 16px 0;
 
             &:before {
-                background-image: url("https://next-geebee.ft.com/image/v1/images/raw/fticon:pullquote?source=o-icons&tint=%23505050,%23505050&format=svg");
+                background-image: url("https://www.ft.com/__origami/service/image/v2/images/raw/fticon:pullquote?source=o-icons&tint=%23505050,%23505050&format=svg");
```
Comment (lucas42) [inline client/src/assets/sass/components/content.scss]: Is the source parameter also wrong here?  Should it be `content-comparator`?

File: client/src/assets/sass/components/lists.scss [hunk 2]
```diff
@@ -288,7 +288,7 @@
     margin-top: -4px;
     height: 9px;
     width: 100%;
-    background: url(https://next-geebee.ft.com/image/v1/images/raw/http%3a%2f%2fnext-geebee.ft.com%2fassets%2fbackgrounds%2ffront-page-section-header-v2.svg?source=next&format=svg&tint=000000,000000) repeat transparent;
+    background: url(https://www.ft.com/__origami/service/image/v2/images/raw/http%3a%2f%2fnext-geebee.ft.com%2fassets%2fbackgrounds%2ffront-page-section-header-v2.svg?source=next&format=svg&tint=000000,000000) repeat transparent;
```
Comment (lucas42) [inline client/src/assets/sass/components/lists.scss]: Any idea where this image comes from?  Does origami have a similar one we can use so we're not depending on next-geebe.ft.com?

**Review by lucas42 — APPROVED** [review]
Sorry, the only person who'd worked on this recently left, so no-one was looking at pull requests.

Yes, all in favour of switching to the new image service.  👍

**Comment by JakeChampion** [comment]
@lucas42 all updated and conflicts resolved. Feel free to merge at your leisure :-)

**Comment by lucas42** [comment]
Thanks!

**Pull request merged**